Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Interop.WPF/Microsoft.Toolkit.Win32.UI.Interop.WPF.csproj

```diff
@@ -0,0 +1,71 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why a separate project? Separate project = Separate DLL = performance overhead. If we're ALWAYS loading this DLL, why not embed its contents?

Reply:
I can't answer this...  My understanding is that these interop assemblies are used elsewhere outside this repo, and they need to be separate to facilitate maintainability wiht the source repo.